app/src/main/java/com/example/voyagerx/adapters/LaunchCarouselAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.voyagerx.adapters
+
+import android.R
+import android.content.Context
+import android.util.Log
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.viewpager.widget.PagerAdapter
+import coil.load
+
+
+class LaunchCarouselAdapter(private val mContext: Context, private val imageList: List<String?>?) : PagerAdapter() {
+
+    override fun getCount(): Int {
+        return imageList?.size ?: 0
+    }
+
+    override fun isViewFromObject(view: View, `object`: Any): Boolean {
```

Comment:
@Celina, what do the ` around object represent (`object`)?

Reply:
The backticks `` can be used to escape Java identifiers. It's used here because the original code I copied from was Java and Android Studio converted the code into Kotlin. If I rename this variable to anything other than `object`, I don't need the backticks. 

I renamed the parameter and removed the backticks since we don't really need it: [c9ce875](https://github.com/onramp-io/twitch-android-project-2/pull/20/commits/c9ce875c940e3cdc3efd49eae69975b717168791)

Kotlin doc: https://kotlinlang.org/docs/java-interop.html#escaping-for-java-identifiers-that-are-keywords-in-kotlin

Reply:
Ooooh, those are backticks. I need to get my glasses prescription updated, haha. Thank you so much for the explanation; that's how you can get around using reserved words - duly noted. Thanks again!!